views/task-list.html

```diff
@@ -0,0 +1,331 @@
+                    <strong class="govuk-tag govuk-tag--grey app-task-list__tag" id="medical-information-status">Not checked</strong>
+                  {% endif %}
+                </li>
+                <li class="app-task-list__item">
+                  <span class="app-task-list__task-name">
+                    <a href="/confirmation-statement/statement-of-capital" aria-describedby="eligibility-status">
+                      Statement of capital
+                    </a>
+                  </span>
+                  {% if statementOfCapital === 'yes' %}
+                    <strong class="govuk-tag app-task-list__tag" id="eligibility-status">
+                      Checked</strong>
+                  {% elseif statementOfCapital === 'no' %}
+                    <strong class="govuk-tag govuk-tag--blue app-task-list__tag" id="eligibility-status">In progress</strong>
+                  {% else %}
+                    <strong class="govuk-tag govuk-tag--grey app-task-list__tag" id="medical-information-status">Not checked</strong>
+                  {% endif %}
+                </li>
+              </ul>
+            </li>
+            <li>
+              <h2 class="app-task-list__section">
+                People
+              </h2>
+              <ul class="app-task-list__items">
+                <li class="app-task-list__item">
+                  <span class="app-task-list__task-name">
+                    <a href="/confirmation-statement/active-officers" aria-describedby="eligibility-status">
+                      Officers
+                    </a>
+                  </span>
+                  {% if taskList.additionalOfficers === 'no' %}
+                    <strong class="govuk-tag app-task-list__tag" id="eligibility-status">
+                      Checked</strong>
+                  {% elseif taskList.additionalOfficers === 'yes' or officers === 'no' or activeOfficers === 'no' %}
+                    <strong class="govuk-tag govuk-tag--blue app-task-list__tag" id="eligibility-status">In progress</strong>
+                  {% else %}
+                    <strong class="govuk-tag govuk-tag--grey app-task-list__tag" id="medical-information-status">Not checked</strong>
+                  {% endif %}
+                </li>
+                <li class="app-task-list__item">
+                  <span class="app-task-list__task-name">
+                    {% if tradingStatus === '1' %}
+                      <a href="/confirmation-statement/active-pscs" aria-describedby="read-declaration-status">
+                        People with significant control (PSC)
+                      </a>
+                    </span>
+                    {% if taskList.pscStatement === 'yes' %}
+                      <strong class="govuk-tag app-task-list__tag" id="eligibility-status">
+                        Checked</strong>
+                    {% elseif taskList.additionalPscs === 'yes' or  taskList.pscs === 'no' or activePscs === 'no' %}
```

Comment:
the multiple tasklist checks should be removed, we should only pass a flag in to the template to toggle the status of a task